Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -107,6 +107,34 @@ The following attributes can be added to the `<Provider />` element to configure
 
 * __`NumStorageGrains="10"`__ (optional) - The number of grains to use to store the state, defaults to `10`
 
+### ADO.NET Persistence
+
+Initializing the database structures needs table named `Storage` from the database scripts and `OrleansQuery` with its contents (or the queries that use `Storage` table).
```

Comment:
I would remove this line, it's an implementation detail which might be mean something to advanced users (maybe add it to the rationale)